src/Flatfile.ts

```diff
@@ -272,6 +272,7 @@ export class Flatfile extends TypedEventManager<IEvents> {
 
 export const SESSION_CONFIG_KEYS: (keyof DataReqOptions)[] = [
   'autoContinue',
+  'customFields',
```

Comment:
we still need to pass this to `openInEmbeddedIframe` `openInNewWindow`
![image](https://user-images.githubusercontent.com/47579072/176318157-553aa6de-18fb-421d-ade5-ddcc31ee94de.png)